src/renderer/user-styles-component.ts

```diff
@@ -24,8 +24,12 @@ export const userStylesComponent = Vue.extend({
 
             --user-input--background-color: {{ colortheme.userInputBackgroundColor }};
             --user-input--color: {{ colortheme.userInputTextColor }};
+            --user-input--radius: {{appearance.userInputBorderRadius}};
+            --user-input--bottom-margin: {{appearance.userInputBottomMargin}}px;
+
 
             --search-results--background-color: {{ colortheme.searchResultsBackgroundColor }};
+            --search-results--radius: {{appearance.searchResultsBorderRadius}};
```

Comment:
I suggest that you rename this css variable to `--search-results--border-radius`.